scylladb/sphinx-scylladb-theme
# feat: add mermaid.js support

Adds [Mermaid.js](https://mermaid.js.org/) support.

## How to test

1. Clone the PR and preview the docs.
2. Open http://127.0.0.1:5500/examples/diagrams/
3. Verify the sample diagram renders as expected:

    ![image](https://github.com/user-attachments/assets/1c82bd3d-db6e-46d4-8fc0-08c472425936)
